### PR TITLE
ui: tweak capacity tooltip (spaces were missing)

### PR DESCRIPTION
**Pull request by vilterp**
Before:
screen shot 2018-01-17 at 12 52 03 pm

After:
screen shot 2018-01-17 at 12 53 27 pm

### CODE REVIEW

**Comment by kuanluo** [comment]
Small nit: Is the indentation in the paragraph after "capacity" intentional? It looks more accidental because of the space...

**Comment by vilterp** [comment]
Yeah, all three paragraphs are indented. I think it looks decent…

screen shot 2018-01-17 at 1 18 16 pm

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/overview.tsx [hunk 1]
```diff
@@ -85,15 +85,15 @@ export default function (props: GraphDashboardProps) {
             <dd>
               Total disk space available {tooltipSelection} to CockroachDB.
                 <em>
-                Control this value per node with the
-                  <code>
+                Control this value per node with the{" "}
```
Comment (couchand) [inline pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/overview.tsx]: as an aside, stylistically I'd put the spaces on their own line

**Review by couchand — APPROVED** [review]
LGTM, thanks for fixing.

File: pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/overview.tsx [hunk 2]
```diff
@@ -85,15 +85,15 @@ export default function (props: GraphDashboardProps) {
             <dd>
               Total disk space available {tooltipSelection} to CockroachDB.
                 <em>
```
Comment (couchand) [inline pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/overview.tsx]: maybe add one above all the `<em>`s, too?

**Comment by couchand** [comment]
@kuanluo I would definitely be in favor of revisiting this tooltip to see how we can make it more effective and generalized.  It seems like details about each series is quite common, and tying it more clearly to the series lines themselves and making it apply to other charts would be a good move.

**Comment by kuanluo** [comment]
I was going to suggest making the indentation a bit more obvious would be a great small step.

**Pull request merged**